contrib/completions/bash/buildah

```diff
@@ -520,6 +520,23 @@ return 1
      esac
  }
 
+ _buildah_inspect() {
+     local options_with_args="
+       --format
+       -f
+       --type
```

Comment:
Makes sense, though I'll have to look that one up.  We'll probably want that for a --format option later anyway.